src/Entity/Adherent.php

```diff
@@ -1198,7 +1198,7 @@ public function setChartAccepted(bool $chartAccepted): void
         $this->chartAccepted = $chartAccepted;
     }
 
-    public function getMandate(): ?string
+    public function getMandate(): ?array
```

Comment:
IMO we can now rename `getMandate()` to `getMandates()`, WDYT ?

Reply:
:heavy_check_mark: